src/main/java/net/openhft/chronicle/wire/VanillaMessageHistory.java

```diff
@@ -53,6 +58,7 @@ public class VanillaMessageHistory extends SelfDescribingMarshallable implements
     private int sources;
     private int timings;
     private boolean addSourceDetails = false;
+    private boolean wallClock = HISTORY_WALL_CLOCK;
```

Comment:
should be final?

Reply:
Do we actually need `wallClock` field - maybe all we need is a `LongSupplier`  `wallClock ? CLOCK.currentTimeNanos() : System.nanoTime();`